pkg/processor/trigger/http/types.go

```diff
@@ -48,7 +52,34 @@ func NewConfiguration(ID string,
 	}
 
 	if newConfiguration.ReadBufferSize == 0 {
-		newConfiguration.ReadBufferSize = 16 * 1024
+		newConfiguration.ReadBufferSize = DefaultReadBufferSize
+	}
+
+	if newConfiguration.CORS.Enabled {
+
+		// take defaults
+		_cors := cors.NewCORS()
```

Comment:
1. `corsInstance` (loose convention)
2. move to a `createCORS` helper function - it pollutes the constructor